health/notifications/awssns/metadata.yaml

```diff
@@ -0,0 +1,123 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-awssns'
+  meta:
+    name: 'AWS SNS'
+    link: 'https://aws.amazon.com/sns/'
+    categories:
+      - notify.agent
+    icon_filename: 'awssns.png'
+  keywords:
+    - AWS SNS
+  overview:
+    notification_description: |
+      As part of its AWS suite, Amazon provides a notification broker service called 'Simple Notification Service' (SNS). Amazon SNS works similarly to Netdata's own notification system, allowing to dispatch a single notification to multiple subscribers of different types. Among other things, SNS supports sending notifications to:
+      - Email addresses
+      - Mobile Phones via SMS
+      - HTTP or HTTPS web hooks
+      - AWS Lambda functions
+      - AWS SQS queues
+      - Mobile applications via push notifications
+      You can send notifications through Amazon SNS using Netdata's Agent alert notification feature, which supports dozens of endpoints, user roles, and more.
+    notification_limitations: |
+      - While Amazon SNS supports sending differently formatted messages for different delivery methods, Netdata does not currently support this functionality.
+      - For email notification support, we recommend using Netdata's email notifications, as it is has the following benefits:
+        - In most cases, it requires less configuration.
+        - Netdata's emails are nicely pre-formatted and support features like threading, which requires a lot of manual effort in SNS.
+        - It is less resource intensive and more cost-efficient than SNS.
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - The [Amazon Web Services CLI tools](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) (awscli).
+            - An actual home directory for the user you run Netdata as, instead of just using `/` as a home directory. The setup depends on the distribution, but `/var/lib/netdata` is the recommended directory. If you are using Netdata as a dedicated user, the permissions will already be correct.
+            - An Amazon SNS topic to send notifications to with one or more subscribers. The Getting Started section of the Amazon SNS documentation covers the basics of how to set this up. Make note of the Topic ARN when you create the topic.
+            - While not mandatory, it is highly recommended to create a dedicated IAM user on your account for Netdata to send notifications. This user needs to have programmatic access, and should only allow access to SNS. For an additional layer of security, you can create one for each system or group of systems.
+            - Terminal access to the Agent you wish to configure.
+    configuration:
+      file:
+        name: 'health_alarm_notify.conf'
+      options:
+        description: 'The following options can be defined for this notification'
+        folding:
+          title: 'Config Options'
+          enabled: true
+        list:
```

Comment:
We also have an option not cited in this file:

```sh
# The full path of the aws command.
# If empty, the system $PATH will be searched for it.
# If not found, Amazon SNS notifications will be silently disabled.
aws=""
```

I think it is a good idea to cite it.